ClassicAssist/Data/Options.cs

```diff
@@ -26,8 +26,8 @@ public class Options : SetPropertyNotifyChanged
         private bool _abilitiesGump = true;
         private int _abilitiesGumpX = 100;
         private int _abilitiesGumpY = 100;
-        private bool _actionDelay;
-        private int _actionDelayMs;
+        private bool _actionDelay =  true;
```

Comment:
This doesn't do anything, the default is already true if creating a new profile...
```c#
            Options.ActionDelay = general["ActionDelay"]?.ToObject<bool>() ?? true;
            Options.ActionDelayMS = general["ActionDelayMS"]?.ToObject<int>() ?? 1000;